src/taco-utils/tacoPackageLoader.ts

```diff
@@ -93,13 +93,13 @@ module TacoUtility {
     }
 
     export class TacoPackageLoader {
+        public static FILE_REGEX_PREFIX: string = "file://";
```

Comment:
@MSLaguana I know why this needs to be changed but instead of making it public, wouldn't it be better to roll this into a public API or may be add a tslint exception